src/com/organizationmanager/Coordinates.java

```diff
@@ -0,0 +1,15 @@
+package com.organizationmanager;
+
+public class Coordinates {
+    private float x; // Max value: 84
+    private Integer y; // Max value: 239, cannot be null

```

Comment:
x can be null, why is it primitive?

y can not be null, why is it boxed integer?